akka-docs/rst/java/persistence.rst

```diff
@@ -859,7 +876,11 @@ Local snapshot store
 --------------------
 
 Local snapshot store plugin config entry is ``akka.persistence.snapshot-store.local`` and it writes snapshot files to
-the local filesystem. The default storage location is a directory named ``snapshots`` in the current working
+the local filesystem. Enable this plugin by defining config property:
```

Comment:
Maybe add an explicit note that it's OK to run without a snapshot store configured even if you use persistence, until you actually call saveSnapshot somewhere (and that cluster sharding will use snapshotting?).